spring-rest/pom.xml

```diff
@@ -325,7 +325,8 @@
 		<!-- Maven plugins -->
 		<cargo-maven2-plugin.version>1.6.0</cargo-maven2-plugin.version>
 		<findbugs-maven-plugin.version>3.0.4</findbugs-maven-plugin.version>
-
+		<checkstyle-maven-plugin.version>3.0.0</checkstyle-maven-plugin.version>
+		<dependency.locations.enabled>false</dependency.locations.enabled>
```

Comment:
Added this line so that build does not take a long time. If we remove this the following issue will happen:
https://stackoverflow.com/questions/26144541/why-am-i-getting-maven-error-unable-to-determine-if-resource-x-exists-in-http